zulip_bots/zulip_bots/bot_shell.py

```diff
@@ -32,15 +32,21 @@ def parse_args():
     return args
 
 
-def main():
+def main() -> None:
     args = parse_args()
 
-    bot_path, bot_name = resolve_bot_path(args.bot)
+    # NOTE: Use of only this implies bots from eg. registry cannot be explored in this way
```

Comment:
Let's open an issue for fixing this to support everything that the other bot interfaces support.